vizro-ai/docs/pages/user-guides/genAI-use-case.md

```diff
@@ -0,0 +1,105 @@
+# How to add Vizro-AI created charts into a Vizro dashboard
```

Comment:
Would it be possible to rename the file to make it more obvious what this page explains something like "add-generated-chart-usecase.md" perhaps?